.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+before_script: "npm run build"
+script:
+  - "npm test"
+  - "npm run lint"
```

Comment:
`sudo: false` is enabled by default: http://docs.travis-ci.com/user/workers/container-based-infrastructure/
> For repos we recognize on or after 2015-01-01, linux builds are sent to our container-based infrastructure.